analyzer/server.go

```diff
@@ -207,6 +221,12 @@ func (s *Server) RegisterRpcEndpoints() {
 			"/rpc/flows",
 			s.FlowSearch,
 		},
+		{
+			"LayerConversation",
+			"GET",
+			"/data/conversation.json/{layer}",
```

Comment:
Could gorilla parse that in the middle ?
Is it REST compliant ?
It's not rpc either ...